### ISSUE TITLE
Win10 stable + pwsh.exe + ssh.exe + mouse events?

### ISSUE THREAD

**Issue by colemickens** [Microsoft/console#361]
This bug-tracker is monitored by Windows Console development team and other technical types.  **We like detail!**

If you have a feature request, please post to [the UserVoice](https://wpdev.uservoice.com/forums/266908).  

> **Important: When reporting BSODs or security issues, DO NOT attach memory dumps, logs, or traces to Github issues**. Instead, send dumps/traces to secure@microsoft.com, referencing this GitHub issue.

Please use this form and describe your issue, concisely but precisely, with as much detail as possible

* Your Windows build number:  ```Microsoft Windows [Version 10.0.17763.292]```

* What you're doing and what's happening: (Copy & paste specific commands and their output, or include screen shots)

Using the latest ssh.exe release (from GitHub as it's usually a bit newer than what's included in Windows 10 builds).

SSH into a remote host. Use tmux/neovim/vim, try to use the mouse in places where it works under usual Linux terminal emulators.

* What's wrong / what should be happening instead:

This is probably user error on my part, or possibly a feature request, but I can't figure out how to get mouse events to work in this scenario. I feel like it's worked in past incarnations of this computer's windows 10 installs... but maybe that was via WSL?

tl;dr: how can I use my mouse with vim on a remote host using pwsh.exe + ssh.exe from a stable Win10 client?

Thanks!

**Comment by DHowett-MSFT** [comment]
Hey @colemickens! Thanks for the report.
Since you're SSHing to a remote Linux host (I presume based on "places where it works under usual Linux terminal emulators",) that'll be due to PowerShell/Win32-OpenSSH#1310. In short, Win32-OpenSSH is doing its own input translation and is missing out on Console's built-in mouse support.

If you're accessing a Windows host, it's more likely to be because our pty infrastructure doesn't support translating incoming mouse events. If that's the case, I'm happy to repoen this, but we do already have a workitem tracking that on our side. :smile:

/cc @manojampalam for the Win32-OpenSSH issue.

**Comment by DHowett-MSFT** [comment]
Notably, using ssh from within WSL will work for mousing to a remote Linux host.

**Comment by colemickens** [comment]
Thank you for the detailed answer @DHowett-MSFT and the cross-references to track. Cheers!
